samples/build.gradle

```diff
@@ -52,8 +52,10 @@ dependencies {
     kapt "androidx.lifecycle:lifecycle-compiler:$lifecycle_version"
 
     // Glide
-    implementation "com.github.bumptech.glide:glide:$versions.glideVersion"
-    implementation "com.github.florent37:glidepalette:$versions.glidePaletteVersion"
+    implementation ("com.github.bumptech.glide:glide:$versions.glideVersion")
+    implementation ("com.github.florent37:glidepalette:$versions.glidePaletteVersion"){
```

Comment:
We may completely delete this left over dependency and its version.

Reply:
Sure! I didn't remove it because I thought you're using it